morph/utils.js

```diff
@@ -203,6 +203,8 @@ export const getAllowedStyleKeys = node => {
     ]
   } else if (node.action || isTable(node) || getActionableParent(node)) {
     return ['base', 'isFocused', 'isHovered', 'isDisabled', 'print']
+  } else if (node.goTo) {
```

Comment:
I think it should go on the line above, as disabled is also an option for links

Reply:
is it though? It's not listed here - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Attributes

Reply:
I guess you're right. Ah well, let's go with it as is :)